collections-constants.go

```diff
@@ -0,0 +1,24 @@
+package shopify
+
+const (
+	// AlphaAsc - Alphabetically, in ascending order (A - Z).
+	AlphaAsc = "alpha-asc"
+	// AlphaDesc - Alphabetically, in descending order (Z - A).
+	AlphaDesc = "alpha-desc"
+	// BestSelling - By best-selling products.
+	BestSelling = "best-selling"
+	// Created - By date created, in ascending order (oldest - newest).
+	Created = "created"
+	// CreatedDesc - By date created, in descending order (newest - oldest).
+	CreatedDesc = "created-desc"
+	// Manual - In the order set manually by the shop owner.
+	Manual = "manual"
+	// PriceAsc - By price, in ascending order (lowest - highest).
+	PriceAsc = "price-asc"
+	// PriceDesc - By price, in descending order (highest - lowest).
+	PriceDesc = "price-desc"
+	// Web - The collection is published to the Online Store channel but not published to the Point of Sale channel.
+	Web = "web"
+	// Global - The collection is published to both the Online Store channel and the Point of Sale channel.
+	Global = "global"
```

Comment:
These should be grouped by the field they are for and also contain more detail in the var names for the type and field they are for, follow the order-constants.go example